theano/gpuarray/tests/test_linalg.py

```diff
@@ -112,3 +115,75 @@ def test_linalgerr_solve(self):
 
         fn = theano.function([A, b], [solver], mode=mode_with_gpu)
         self.assertRaises(LinAlgError, fn, A_val, x_val)
+
+
+class TestMagma(unittest.TestCase):
+    def test_gpu_matrix_inverse(self):
```

Comment:
Make the tests skipped if magma isn't there.

Add a test for the inplace matrix_inverse